### PR TITLE
salt: Add `bash-completion` for `kubectl`

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
**Component**:

'salt', 'kubectl'

**Context**: 

#2996 

**Summary**:

Add a `/etc/bash_completion.d/kubectl` file for completion in bash for
kubectl command

---

Fixes: #2996

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@
 
 - Bump etcd version to 3.4.13-0 (PR[#3008](https://github.com/scality/metalk8s/pull/3008))
 
+- [#2996](https://github.com/scality/metalk8s/issues/2996) - The `bash-completion`
+  completions for the `kubectl` command are now setup on every master nodes
+  (PR[#3039](https://github.com/scality/metalk8s/pull/3039))
```
Comment (NicolasT) [inline CHANGELOG.md]: I noticed this before, shouldn't we leave a space between `PR` and the `#123`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,10 @@
 
 - Bump etcd version to 3.4.13-0 (PR[#3008](https://github.com/scality/metalk8s/pull/3008))
 
+- [#2996](https://github.com/scality/metalk8s/issues/2996) - The `bash-completion`
+  completions for the `kubectl` command are now setup on every master nodes
```
Comment (NicolasT) [inline CHANGELOG.md]:
```suggestion
  completions for the `kubectl` command are now provided when `kubectl` is installed
```

File: salt/metalk8s/kubectl/configured.sls [hunk 3]
```diff
@@ -0,0 +1,14 @@
+include:
+  - .installed
+
+Create kubectl bash completion file:
+  file.managed:
+    - name: /etc/bash_completion.d/kubectl
+    - contents: __slot__:salt:cmd.run('kubectl completion bash')
+    - makedirs: True
+    - dir_mode: 755
+    - mode: 755
```
Comment (NicolasT) [inline salt/metalk8s/kubectl/configured.sls]: Is this required? I think `644` should do?

File: CHANGELOG.md (hunk 1 above)
Comment (alexandre-allard) [reply CHANGELOG.md]: We should.

Comment (TeddyAndrieux) [reply CHANGELOG.md]: Hmm don't really know, I just saw this as well some changelog entry have a space other do not have... 
I don't know

File: salt/metalk8s/kubectl/configured.sls (hunk 3 above)
Comment (TeddyAndrieux) [reply salt/metalk8s/kubectl/configured.sls]: Oh yes MB just copy-pasted the dir mode but it's wrong indeed

File: CHANGELOG.md (hunk 1 above)
Comment (TeddyAndrieux) [reply CHANGELOG.md]: Ok I will change this one and will update other ones in another PR

#3040

**Review by NicolasT — APPROVED** [review]

**Pull request merged**